plugins/auth-backend/src/service/router.ts

```diff
@@ -81,16 +86,27 @@ export async function createRouter(
   });
   const keyDurationSeconds = BACKSTAGE_SESSION_EXPIRATION;
 
-  const tokenIssuer = new TokenFactory({
-    issuer: authUrl,
-    keyStore,
-    keyDurationSeconds,
-    logger: logger.child({ component: 'token-factory' }),
-    algorithm:
-      tokenFactoryAlgorithm ??
-      config.getOptionalString('auth.identityTokenAlgorithm'),
-  });
-
+  let tokenIssuer: TokenIssuer;
+  const mountedTokenIssuerConfig = config.getOptional<MountedTokenIssuerConfig>(
+    'auth.issuer.mountedTokenIssuer',
```

Comment:
You'll want to update the `config.d.ts` file at the root of this plugin with this new configuration

Mode details on this are here: https://backstage.io/docs/conf/defining

Reply:
Good one! I always forget about that file. I've updated it!